src/acrcssc/setup.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+
+VERSION = '1.1.1'
+
```

Comment:
Please adjust the initial version for `acrcssc` to be `1.0.0b1`, instead of `1.1.1`

For more info about the extension versioning schema, please refer to doc here:https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#initialization